tests/integrations/stdlib/test_httplib.py

```diff
@@ -106,3 +106,5 @@ def test_httplib_misuse(sentry_init, capture_events):
         "status_code": 200,
         "reason": "OK",
     }
+
+    conn.close()
```

Comment:
I think this is better to use the `request` fixture and `request.addfinalizer(conn.close)` right after the conn was created on line 84.  That way this gets cleaned up even when then test fails

Reply:
Oh, good idea, thanks. Done.